clipboard-apis/async-html-script-removal.https.html

```diff
@@ -31,9 +31,9 @@
 const html_with_script =
   '<title>Title of the document</title> <script>const a = 5;</scr'
   + 'ipt> <p>Hello World</p>';
-const html_without_script =
-  '<title>Title of the document</title> <p>Hello World</p>';
-promise_test(async t => {
+const html_script = '<script>const a = 5;</scr'
+  + 'ipt>';
```

Comment:
why this weird breaking?

Reply:
I didn't the original way of writing the script. But this is part of an old technique to avoid parsing error of script inside HTML.

Reply:
aaah interesting!

Reply:
made it a bit clearer and aligned with the previous version.